internal/dataplane/parser/failures_test.go

```diff
@@ -0,0 +1,82 @@
+package parser_test
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/dataplane/parser"
+	kongv1 "github.com/kong/kubernetes-ingress-controller/v2/pkg/apis/configuration/v1"
+)
+
+const someValidTranslationFailureReason = "some valid reason"
+
+var someValidTranslationFailureCausingObjects = []client.Object{&kongv1.KongIngress{}, &kongv1.KongPlugin{}}
```

Comment:
Why not put this into the test itself ?

Reply:
They're used both in `TestTranslationFailure` and `TestTranslationFailuresCollector`.

Reply:
Ok got it, I missed it. In that case I'd suggest to make a `func` returning those. Better that than be surprised why something manipulated those and get surprising results in tests.

Reply:
Yeah, fair point: 9f916d4 :)